oauth_tokens/base.py

```diff
@@ -81,7 +81,7 @@ def add_data_credentials(self, data):
         pass
 
     def get_form_data_from_content(self, content, **kwargs):
-        bs = BeautifulSoup(content)
+        bs = BeautifulSoup(content, 'html5lib')
```

Comment:
I am not sure, because I am using it only for twitter. What made me change every part was warnings from django.